go-controller/pkg/ovn/egressgw.go

```diff
@@ -205,7 +205,7 @@ func (oc *Controller) addPodExternalGWForNamespace(namespace string, pod *kapi.P
 				"IP address as pod: %q. IP address: %q", namespace, pod.Name, podName, gwIP.String())
 		}
 	}
-	nsInfo.routingExternalPodGWs[pod.Name] = egress
+	nsInfo.routingExternalPodGWs[fmt.Sprintf("%s_%s", pod.Namespace, pod.Name)] = egress
```

Comment:
@trozet can we make a function for `makePodGWKey(pod *v1.Pod) string` ?

Reply:
sure